src/Pages/URLShortener/URLShortener.js

```diff
@@ -107,6 +117,24 @@ export default function URLShortenerPage(props) {
     }
   }
 
+  function handleSortUrls(columnName) {
+    if (columnName === null) {
+      return;
+    }
+    if(currentSortColumn === columnName) {
+      if (currentSortOrder === 'ASC') {
+        setCurrentSortOrder('DESC');
+      } else if (currentSortOrder === 'DESC') {
+        setCurrentSortOrder('none');
```

Comment:
how about instead of `'none'` we just set to `null`?

Reply:
resolved, changed logic to set column to null as well